src/stories/standalone/UIKit/InputControls/DateTimeInput.story.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { boolean, text } from "@storybook/addon-knobs";
+import moment, { Moment } from "moment";
+import { action } from "@storybook/addon-actions";
+import DateTimeInput from "../../../../standalone/UIKit/InputControls/DateTimeInput";
+import { MaterialUiPickersDate } from "@material-ui/pickers/typings/date";
+
+export const DateTimeInputStory = (): React.ReactElement => {
+	const [selectedDate, setSelectedDate] = useState(moment());
```

Comment:
why are you using moment here? This is a normal Date object or null. `useState<MaterialUiPickersDate>(null)`

Reply:
Updated